src/main/java/com/smartling/marketo/sdk/MarketoClientManager.java

```diff
@@ -18,4 +20,6 @@ public interface MarketoClientManager {
     MarketoProgramClient getMarketoProgramClient();
 
     MarketoTokenClient getMarketoTokenClient();
+
+    MarketoTriggerCampaignClient getMarketoTriggerCampaignClient();
```

Comment:
Naming convention of the Marketo and this sdk is not followed. I suggest to rename this client to SmartCampaignClient. It will be able to be extended with other smart campaign endpoints.